infra/modules/s3/vars.tf

```
@@ -1,3 +1,11 @@
 variable "environment" {
   type = string
 }
+
+variable "pub_router_failure_channel" {
+  type = string
+}
+
+variable = "pub_router_api_key" {
```

```suggestion
variable "pub_router_api_key" {
```

This equals sign will cause a syntax error.